boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -62,6 +62,11 @@
 #define MCUBOOT_ENCRYPT_RSA
 #endif
 
+#ifdef CONFIG_BOOT_ENCRYPT_ECIES
```

Comment:
There is a bit of a potential of confusion with this name. Unlike RSA-OEP, ECIES was designed to encrypt (and protect) arbitrary data. When used that way, it offers additional security properties over RSA-OEP. When used as a keywrap, it doesn't offer these properties. However, the name "ENCRYPT_ECIES" kind of suggests we are encrypting the data itself using ECIES.

Perhaps we should call this ENCRYPT_ECIES_KW (or keywrap). Or just use the EC256 that we use elsewhere, which doesn't really mean anything by itself. "ENCRYPT_RSA" doesn't really mean anything by itself, so is less likely to cause confusion.

Reply:
I did not do exactly that, but I think it is properly described in the encrypted images doc now.